### ISSUE TITLE
misc.but reports false positive

### ISSUE THREAD

**Issue by petersohn** [amperser/proselint#745]
Run proselint on the following file:

```
This line is OK,
but this is not.
```

The following error is reported:

```
misc.but No paragraph should start with a 'But'.
```

That "but" is not only not at the beginning of the paragraph, it's not even at the beginning of a sentence.
